<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPassWindow</class>
 <widget class="QWidget" name="ResetPassWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="BackGroundFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>430</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="TopFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>380</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.477273, x2:1, y2:0.471591, stop:0 rgba(80, 174, 247, 255), stop:1 rgba(15, 143, 242, 255));
border: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ResetNameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
font-size:18px;</string>
     </property>
     <property name="text">
      <string>修改密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CloseButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../statics/imgs/close.png</normaloff>../statics/imgs/close.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="BodyFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>380</width>
      <height>380</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="TitleLabel">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>20</y>
       <width>221</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font-size:18px;</string>
     </property>
     <property name="text">
      <string>国联恶意代码辅助监测系统</string>
     </property>
    </widget>
    <widget class="QLabel" name="VersionLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>54</y>
       <width>66</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(43, 128, 247);
background-color: rgb(233, 242, 254);
border-radius: 13;
font-size:14px;</string>
     </property>
     <property name="text">
      <string>   主机版</string>
     </property>
    </widget>
    <widget class="QFrame" name="OldPassFrame">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>94</y>
       <width>301</width>
       <height>36</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 245, 245);
border: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="OldPassInput">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>261</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
color: black;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>旧密码</string>
      </property>
     </widget>
     <widget class="QLabel" name="OldPassIcon">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>7</y>
        <width>20</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../statics/imgs/passinput.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="NewPassFrame">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>150</y>
       <width>301</width>
       <height>36</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 245, 245);
border: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="NewPassInput">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>261</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
border: none;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>新密码</string>
      </property>
     </widget>
     <widget class="QLabel" name="NewPassIcon">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>7</y>
        <width>20</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../statics/imgs/passinput.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="ConfirmButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>320</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size:16px;
	border: none;
	border-radius: 15;
	background-color: rgb(43, 128, 247);
}
:hover{
    font-size:16px;
	background-color: rgb(146, 212, 255);
}


</string>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CancelButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>320</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size:16px;
	border: none;
	background-color: rgb(246, 41, 41);	
	border-radius: 15;
}
:hover{
    font-size:16px;
	background-color: rgb(255, 188, 226);
}

</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QFrame" name="ConfirmPassFrame">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>207</y>
       <width>301</width>
       <height>36</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 245, 245);
border: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="ConfirmPassInput">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>261</width>
        <height>36</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;
border: none;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>确认密码</string>
      </property>
     </widget>
     <widget class="QLabel" name="ConfirmPassIcon">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>7</y>
        <width>20</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../statics/imgs/passinput.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font-size:13px;</string>
     </property>
     <property name="text">
      <string>新密码至少8个字符,其中数字至少1个,大写字母至少1个</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font-size:13px;</string>
     </property>
     <property name="text">
      <string>小写字母至少1个，特殊字符至少1个</string>
     </property>
    </widget>
    <widget class="QLabel" name="MessageLabel1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
font-size:13px;</string>
     </property>
     <property name="text">
      <string>MessageLabel1</string>
     </property>
    </widget>
    <widget class="QLabel" name="MessageLabel2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
font-size:13px;</string>
     </property>
     <property name="text">
      <string>MessageLabel2</string>
     </property>
    </widget>
    <widget class="QLabel" name="MessageLabel3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
font-size:13px;</string>
     </property>
     <property name="text">
      <string>MessageLabel3</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
